### PR TITLE
Add imprint URL to Template.pageFooter

### LINKED ISSUE #16965: Add imprint URL to Template.pageFooter
Hi,

would you accept a pull request that extends on PR https://github.com/matomo-org/matomo/pull/13219 to add an Imprint URL to `Template.pageFooter` and settings?

Reasoning: Most sites I know of have distinct pages for data privacy policy, terms & conditions and the imprint and right now there is no way to link an imprint which is required by some German law.  
The issue this PR was created from (#12938) and its duplicate #7513 actually mention the imprint but there is no reason stated why it was omitted.

This would be my first contribution to Matomo. Is there anything special I should bear in mind when implementing this?

### PR DESCRIPTION
**Pull request by wedi**
### Description:

This pull request is a follow up on PR #13219 and adds an imprint URL to the page footer.

Fixes: #16965

### Review

* [ ] Functional review done
* [ ] Usability review done (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] Security review done [see checklist](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] Code review done
* [ ] Tests were added if useful/possible _- but not tested, see above_
* [ ] Reviewed for breaking changes
* [ ] Developer changelog updated if needed
* [ ] Documentation added if needed
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by wedi** [comment]
My first idea was to switch to a multituple (label and URL) to handle future requests for more URLs, too, but in the end I went with this simple solution as I couldn't deduct a way to handle label translations from the existing code I read.

File: plugins/PrivacyManager/lang/de.json [hunk 1]
```diff
@@ -175,11 +175,14 @@
...
         "PrivacyPolicyUrl": "URL für Datenschutzrichtlinie",
         "PrivacyPolicyUrlDescription": "Ein Link zu Ihrer Datenschutzrichtlinienseite.",
+        "ImprintUrl": "URL für Impressum",
+        "ImprintUrlDescription": "Ein Link zu Ihrem Impressum.",
         "TermsAndConditionUrl": "URL für Geschäftsbedingungen",
         "TermsAndConditionUrlDescription": "Ein Link zu Ihrer Geschäftsbedingungenseite.",
         "PrivacyPolicyUrlDescriptionSuffix": "Wenn Sie etwas angeben, wird es am Ende der Anmeldeseite und auf Seiten angezeigt, auf die der '%1$s' Benutzer Zugriff hat.",
         "ShowInEmbeddedWidgets": "In eingebetteten Widgets anzeigen",
         "ShowInEmbeddedWidgetsDescription": "Wenn angewählt, wird am Ende der eingebetteten Widgets ein Link zu Ihren Datenschutzrichtlinien und zu Ihren Geschäftsbedingungen angezeigt.",
+        "Imprint": "Impressum",
```
Comment (diosmosis) [inline plugins/PrivacyManager/lang/de.json:185]: These are added via an automated workflow via transifex and translators that work there. It's only necessary to update the english translation when making pull requests. Can you remove these translations?

Comment (diosmosis) [inline plugins/PrivacyManager/lang/de.json]: These are added via an automated workflow via transifex and translators that work there. It's only necessary to update the english translation when making pull requests. Can you remove these translations?

**Review by diosmosis — CHANGES_REQUESTED** [review]
Works locally, and the code looks good. I left one comment regarding translations, once it's fixed this can be merged. Thanks for the first contribution @wedi!

File: plugins/PrivacyManager/lang/de.json (hunk 1 above)
Comment (wedi) [reply plugins/PrivacyManager/lang/de.json]: Sure. Done.

**Pull request merged**